packages-ts/gauntlet-terra-cw-plus/src/commands/multisig.ts

```diff
@@ -31,6 +31,8 @@ export const wrapCommand = (command) => {
 
     makeRawTransaction = async (signer: AccAddress, state?: State) => {
       const message = await this.command.makeRawTransaction(this.multisig)
+      const estimatedGas = await this.command.simulate(this.multisig, [message])
+      logger.info(`Tx simulation successful: estimated gas usage is ${estimatedGas}`)
```

Comment:
Nice, it was exactly this. 
The message is not entirely accurate though, this tx won't be executed directly, it will be wrapped into the multisig proposal, so the gas usage should be from the multisig tx simulation